ownership-chain/runtime/src/tests.rs

```diff
@@ -61,3 +72,27 @@ fn asset_id_to_address_two_assets_same_owner() {
 		AccountId::from_str("c0f0f4ab324c46e55d02d0033343b4be8a55532d").unwrap()
 	);
 }
+
+#[test]
+fn minimum_balance_should_be_1() {
+	assert_eq!(Balances::minimum_balance(), 0);
+}
+
+#[test]
+fn deposit_minimum_amount_should_succeed() {
+	new_test_ext().execute_with(|| {
+		let alice = AccountId::from_str(ALICE).unwrap();
+		assert_eq!(Balances::total_balance(&alice), 0);
+
+		let minimum_amount = 1;
+
+		let result = Balances::deposit(&alice, minimum_amount, Precision::Exact);
+
+		// I am forced to use match cause result doesn't implement Debug trait
+		match result {
```

Comment:
`assert_ok!()` didn't work?

Reply:
yes it doesn't because it needs  Debug trait to be implemented

Reply:
`assert!(Balances::deposit(&alice, minimum_amount, Precision::Exact).is_ok());`